.github/ISSUE_TEMPLATE/rfc-template.md

```diff
@@ -0,0 +1,112 @@
+---
+name: RFC Template
+about: Fill in this template and you are on your way to submit an RFC to the Project-Origin
+  working group.
+title: "[RFC]000 - Issue title"
+labels: rfc
+assignees: ''
+
+---
+
+---
+name: RFC template
+about: Fill in this template and you are on your way to submit an RFC to the Project Origin working group.
+title: "[RFC]000 - Issue title"
+labels: RFC
+assignees: ''
+---
+# 000-Template
+
+### Note: This template for creating RFCs has been adapted from the template used in the [Energy Track and Trace](https://github.com/Energy-Track-and-Trace) project.
+
+Learn more about [using RFCs in this project](missing link).
```

Comment:
The original project had a pointer to this page I think.
```suggestion
Learn more about [using RFCs in this project](https://github.com/Energy-Track-and-Trace/ett-documentation/tree/main/rfcs).
```
Might not be perfect for us, but it's still a good page so I added it. Maybe we can adapt it for our needs?
Also introducing the way to use this for our local project will need some interactive work I think :)

Reply:
@lenucksi you are absolutely right - some interactive work would be a welcome addition to the template and would ease adoption. 
In addition we might have to consider word templates, hence word is the modus operandi for most non-technical potential commiters.